docs/changelog/0004-add-analytics-queue.md

```diff
@@ -0,0 +1,20 @@
+# 4. Add queue for analytics tasks
+
+**Date:**  2018-08-16
+
+**Optional per env:** Yes
```

Comment:
I think this isn't optional, since the new queue is a required queue in app_processes. If we want to keep it that way other than changing this to no, I think we also need to note that `analytics_queue` needs to be added to the environment.

Reply:
Oops. yeah you're right. I had intially had it as optional because required=False, but changed it due to cal's